rd-kt/rd-framework/src/commonMain/kotlin/com/jetbrains/rd/framework/TaskInterfaces.kt

```diff
@@ -120,10 +121,12 @@ interface IRdEndpoint<TReq, TRes> {
     /**
      * Assigns a handler that executes the API asynchronously.
      */
-    fun set(handler: (Lifetime, TReq) -> RdTask<TRes>)
+    fun set(cancellationAndRequestScheduler: IScheduler? = null, handler: (Lifetime, TReq) -> RdTask<TRes>)
 
     /**
      * Assigns a handler that executes the API synchronously.
      */
-    fun set(handler: (TReq) -> TRes) = set { _, req -> RdTask.fromResult(handler(req)) }
+    fun set(cancellationAndRequestScheduler: IScheduler? = null, handler: (TReq) -> TRes) = set { _, req -> RdTask.fromResult(handler(req)) }
 }
+
+interface ILifetimedRdCallWithEndpoint<TReq, TRes> : ILifetimedRdCall<TReq, TRes>, IRdEndpoint<TReq, TRes>
```

Comment:
Once again. Why do we need such an interface? We can generate RdCall for symmetric generation